### PR TITLE
Only display baseLayers in legend on mobile

### PR DESCRIPTION
**Pull request by oterral**
Wait for deploy of new react-spaital

### CODE REVIEW

File: src/components/Menu/TopicMenu.js [hunk 1]
```diff
@@ -54,6 +71,36 @@ class TopicMenu extends PureComponent {
...
+      layerService.on('change:visible', () => {
+        this.updateCurrentBaseLayerKey();
+      });
+    }
+  }
+
+  unlistenLayerServiceEvent(prevLayerService) {
+    const { layerService } = this.props;
+    if (layerService || prevLayerService) {
+      /* (prevLayerService || layerService).un(
```
Comment (mhaertwig) [inline src/components/Menu/TopicMenu.js]: @oterral  can this be removed?

File: src/components/Menu/TopicMenu.js [hunk 2]
```diff
@@ -126,7 +173,27 @@ class TopicMenu extends PureComponent {
             }}
           />
         </div>
-        <div className="wkp-layer-tree">{layerTree}</div>
+        <div className="wkp-topic-content">
+          {topic.key === activeTopic.key && layerService.getBaseLayers() && (
```
Comment (mhaertwig) [inline src/components/Menu/TopicMenu.js]: Please also hide if there's only one baselayer.

File: src/components/Menu/TopicMenu.js (hunk 1 above)
Comment (oterral) [reply src/components/Menu/TopicMenu.js]: yes

Comment (oterral) [reply src/components/Menu/TopicMenu.js]: or no I fix to unlisten properly.

**Comment by oterral** [comment]
@mhaertwig .  Changes made

**Pull request merged**